packages/libnfs/build.sh

```diff
@@ -3,14 +3,15 @@ TERMUX_PKG_DESCRIPTION="NFS client library"
 TERMUX_PKG_LICENSE="LGPL-2.1, BSD 2-Clause, GPL-3.0"
 TERMUX_PKG_LICENSE_FILE="LICENCE-LGPL-2.1.txt, LICENCE-BSD.txt, LICENCE-GPL-3.txt"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="5.0.3"
+TERMUX_PKG_VERSION="6.0.0"
 TERMUX_PKG_SRCURL=https://github.com/sahlberg/libnfs/archive/libnfs-$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=d945cb4f4c8f82ee1f3640893a168810f794a28e1010bb007ec5add345e9df3e
+TERMUX_PKG_SHA256=6fe64b5a47b2558484c8beb05819c1f1f3e52cc52a7b3a8b805faf398e9a9c24
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_METHOD=repology
 TERMUX_PKG_BREAKS="libnfs-dev"
 TERMUX_PKG_REPLACES="libnfs-dev"
-
-termux_step_pre_configure() {
-	autoreconf -fi
-}
```

Comment:
Does upstream not use autotools anymore, or why is the `autoreconf -fi` no longer necessary?

Reply:
Autoconf bad, CMake good. /j

Actually, it's because Autoconf requires unneeded dependencies while that dependencies only required on specific OS (source: CMakeLists.txt).

Reply:
So we're using CMake instead, thanks.